pkgs/package-json-runner/default.nix

```diff
@@ -0,0 +1,11 @@
+{ jq, js-runtime, writeScript }: writeScript "run-${js-runtime}" ''
+  if start=$(${jq}/bin/jq -r '.scripts.start? // empty' package.json 2>/dev/null); [[ -n $start ]]; then
+    eval $start;
```

Comment:
does bun have an equivalent to `npm run <command>`?

Reply:
yes! good point lmao why would i do this

Reply:
ohhhhhhhh actually i see what's going on here. ok i'm not gonna keep trying this package-json-runner abstraction, that can happen later